packages/images-plugin/test/unit/image-element.spec.js

```diff
@@ -48,7 +48,7 @@ describe('ImageElement', function () {
   describe('.asElement', function () {
     it('should get the webdriver object representation of the element', function () {
       const el = new ImageElement(defTemplate, defRect);
-      el.asElement('ELEMENT').ELEMENT.should.match(/^appium-image-el/);
+      el.asElement().ELEMENT.should.match(/^appium-image-el/);
```

Comment:
should we also verify the w3c element identifier key value is correct? i know this wasn't checked before but seems particularly important now that we only support w3c.

Reply:
changed to use unwrapElement helper